### PR TITLE
Add cloud-provider at creation time

### PR DESCRIPTION
**Pull request by jesusbv**
The cloud provider info is set when updating
but not when registering for the first time.

*Please select the correct option.*

- [X] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

*Please check off each item if the requirement is met.*

- [X] I have verified that my code follows RMT's coding standards with `rubocop`.
- [X] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [X] RMT's test coverage remains at 100%.
- [X] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

### CODE REVIEW

File: lib/rmt.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module RMT
-  VERSION ||= '2.5.8'.freeze
+  VERSION ||= '2.5.9'.freeze
```
Comment (ikapelyukhin) [inline lib/rmt.rb:2]: 2.5.9 is already taken in #569, please use 2.5.10 instead.

**Comment by ikapelyukhin** [comment]
Oh, and package version in `package/obs/rmt-server.spec` also needs to be updated.
BTW, packaging documentation is in [PACKAGE.md](https://github.com/SUSE/rmt/blob/master/PACKAGE.md).

**Comment by thutterer** [comment]
@jesusbv Hey, sorry this PR got no more feedback in such a long time! Are you still interested in getting this merged? There are a few conflicting files now, could you rebase your work on latest master, please? I then would take care that someone from the team reviews this. Thanks!

**Comment by digitaltom** [comment]
@jesusbv please ping us if this pr is still wanted and needs review, so we can help you to get it released.

**Comment by jesusbv** [comment]
> @jesusbv please ping us if this pr is still wanted and needs review, so we can help you to get it released.

Hi @digitaltom, yes I would say it is wanted, currently no cloud-provider info set when registering
thoughts?

**Comment by ngetahun** [comment]
@jesusbv Please update the PR with the PULLREQUEST doc file in `.github`.

**Comment by ngetahun** [comment]
@digitaltom Do we need to add a new db field for the registration proxy in scc to accomodate `cloud_provider` field?

**Comment by digitaltom** [comment]
@ngetahun The cloud provider field is already there in RMT (https://github.com/SUSE/rmt/blob/master/db/schema.rb#L51) and SCC (https://github.com/SUSE/happy-customer/blob/master/glue/db/schema.rb#L111). 
I understand @jesusbv's description like this patch is fixing the setting of it during new registration, while it's already supported and works when a system updates it's hw_info.

File: lib/rmt.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module RMT
-  VERSION ||= '2.5.8'.freeze
+  VERSION ||= '2.5.10'.freeze
```
Comment (ngetahun) [inline lib/rmt.rb:2]: @jesusbv The version bump is `2.5.9`.
```suggestion
  VERSION ||= '2.5.9'.freeze
```

File: package/obs/rmt-server.changes [hunk 3]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jun  5 13:41:29 UTC 2020 - Jesús Bermúdez Velázquez <jesus.bv@suse.com>
+
+- Version 2.5.10
```
Comment (ngetahun) [inline package/obs/rmt-server.changes:4]:
```suggestion
- Version 2.5.9
```

**Review by ngetahun — CHANGES_REQUESTED** [review]

**Review by ngetahun — APPROVED** [review]
LGTM!

**Pull request merged**